tests/network/vmi_multus.go

```diff
@@ -186,13 +186,13 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		Expect(createPtpNetworkAttachmentDefinition(tests.NamespaceTestAlternative, ptpConf2, ptpSubnet)).To(Succeed())
 
 		// Multus tests need to ensure that old VMIs are gone
-		Eventually(func() int {
+		Eventually(func() []v1.VirtualMachineInstance {
```

Comment:
It may be better to return `[]*v1.VirtualMachineInstance`, since we're only after the count anyways, it may save us a bit of copying.

Reply:
I think this comment belongs on the dependent PR #7497 .

Reply:
Yep, thanks